lib/shelley/bench/Restore.hs

```diff
@@ -764,7 +764,7 @@ waitForNodeSync
     => Tracer IO (BenchmarkLog n)
     -> NetworkLayer IO Block
     -> IO SlotNo
-waitForNodeSync tr nw = loop 10
+waitForNodeSync tr nw = loop 120
```

Comment:
I've seen it took up to 15 minutes locally when switching to newer version of cardano-node. Maybe good to be more conservative here?

Reply:
That number works out to 30 minutes, which should be conservative enough, I hope. I added a comment to clarify. It would be clearer if this function used `Control.Retry.retrying` but it doesn't.

Reply:
Hopefully 30 mins is enough (https://input-output-rnd.slack.com/archives/C7PJ29FME/p1616507642042100?thread_ts=1616507577.042000&cid=C7PJ29FME)

Reply:
I think on a dedicated linux benchmark machine it should be fine.